### PR TITLE
Smores

### PR DESCRIPTION
**Pull request by Shweidman**
Added new "smores" path to story.

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/marshmallow.md [hunk 1]
```diff
@@ -177,3 +177,5 @@ Do you:
 [The walls start closing in](malaria/malaria.md)
 
 [Come to Nepal](nepal/nepal.md)
+
+[Try to make smores](smores/smores.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/smores [hunk 2]
```diff
@@ -0,0 +1 @@
+You decide to use your lighter and the marshmallows to make smores! It's too bad you have chocolate...
```
Comment (casey-collab) [inline english/smores]: Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.

Comment (casey-collab) [inline english/smores]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/smores]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**